clar2wasm/src/tools.rs

```diff
@@ -0,0 +1,80 @@
+use clarity::{
+  consts::CHAIN_ID_TESTNET,
+  types::StacksEpochId,
+  vm::{
+      clarity_wasm::initialize_contract,
+      contexts::GlobalContext,
+      costs::LimitedCostTracker,
+      database::{ClarityDatabase, MemoryBackingStore},
+      types::{QualifiedContractIdentifier, StandardPrincipalData},
+      ClarityVersion, ContractContext, Value,
+  },
+};
+
+use crate::{compile, datastore::{StacksConstants, BurnDatastore}};
+
+pub fn evaluate(snippet: &str) -> Option<Value> {
+  let constants = StacksConstants::default();
+  let burn_datastore = BurnDatastore::new(constants);
+  let mut clarity_store = MemoryBackingStore::new();
+  let cost_tracker = LimitedCostTracker::new_free();
+
+  let mut db = ClarityDatabase::new(&mut clarity_store, &burn_datastore, &burn_datastore);
+  db.begin();
+  db.set_clarity_epoch_version(StacksEpochId::latest());
```

Comment:
I'd probably have this passed in, just to avoid needing to think about epoch/version later

Reply:
Good thinking. I've added `evaluate_at` that takes in the epoch and clarity version, and then made `evaluate` use the latest.